index.js

```diff
@@ -212,7 +212,23 @@ function focusTrap(element, userOptions) {
 
     if (!node) {
       throw new Error(
-        "You can't have a focus-trap without at least one focusable element"
+        'Your focus-trap needs to have at least one focusable element'
+      );
+    }
+
+    return node;
+  }
+
+  function getReturnFocusNode(previousActiveElement) {
+    var node = getNodeForOption('setReturnFocus');
+
+    if (!node) {
+      node = previousActiveElement;
+    }
+
+    if (!node || !previousActiveElement) {
+      throw new Error(
+        'Your focus-trap needs to have at least one focusable element'
```

Comment:
Sorry for my previous feedback. this error should tell something else, like "`document.activeElement` or `setReturnFocus` needs to return an element in order for the focus to be returned when focus-trap is deactivated".

But actually I'm not sure if we'd like to throw anything in here. I think in case in `node` is empty, we should just do nothing (then focus won't be returned). I mean, in a rare case, when `document.activeElement` is empty, and somebody programitically activates the focus trap, then if that trap is deactivated, then `document.activeElement` should still be empty, instead of having the app throw a fatal error, right?

Reply:
Yeah, my bad. I used the `getInitialFocusNode()` function as a base to build `getReturnFocusNode`. I simplified the function with the last commit.